.github/workflows/publish-olm-bundle.yaml

```diff
@@ -0,0 +1,79 @@
+name: Submit release OLM bundle
+
+on:
+  release:
+    types:
+      - published
+# Remove this trigger event once workflow behavior is validated
+  issues:
+    types:
+      - opened
+
+jobs:
+  createPullRequest:
+    name: Publish new OperatorHub release
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Determine Repository Variable
+      # Remove this conditional logic once the workflow behavior
+      # is validated
+          run: |
+            if [[ "${{ github.event_name }}" == "issues" && "${{ github.event.action }}" == "opened" ]]; then
+              echo "repository=k8s-operatorhub/community-operators" >> $GITHUB_ENV
+            else
+              echo "repository=fmuyassarov/community-operators" >> $GITHUB_ENV
+
```

Comment:
Also, I think there is a missing terminating `fi` for the shell `if` here...

Reply:
oh my bad